### PR TITLE
Functionality/hydration dom

### PR DESCRIPTION
**Pull request by Jedeo**
What’s this PR do?
- Adds Hydration data to the DOM
Where should the reviewer start?
-see `index.html, stylesheet , script.js , and user.js`
How should this be manually tested?
- Tested Using console logs and the Browser

### CODE REVIEW

File: src/css/styles.css [hunk 1]
```diff
@@ -115,6 +116,34 @@ main {
...
+.hydra-chart{
+  
+  font-size: 1.7vw;
+  width: 28vw;
+  margin-top: 32vh;
+  position: absolute;
+}
+.hydration-container {
+ height: 75vh;
+ width: 30vw;
```
Comment (piperjarrett) [inline src/css/styles.css:139]: We might change these to px, just to stay consistent!

File: src/scripts.js [hunk 2]
```diff
@@ -143,92 +153,150 @@ function displaySleepForAWeek() {
...
+    <td class="sleep-data">${sleepInAWeek[5].hoursSlept}</td>
+    <td class="sleep-data">${sleepQualityInAWeek[5].sleepQuality}</td>
   </tr>
   <tr>
-    <td>${sleepInAWeek[6].date}</td>
-    <td>${sleepInAWeek[6].hoursSlept} hours</td>
-    <td>${sleepQualityInAWeek[6].sleepQuality}</td>
+    <td class="sleep-data">${sleepInAWeek[6].date}</td>
+    <td class="sleep-data">${sleepInAWeek[6].hoursSlept} hours</td>
+    <td class="sleep-data">${sleepQualityInAWeek[6].sleepQuality}</td>
```
Comment (piperjarrett) [inline src/scripts.js:196]: I like that you added classes!

File: src/scripts.js [hunk 3]
```diff
@@ -143,92 +153,150 @@ function displaySleepForAWeek() {
...
+  <tr>
+    <td class="hydra-data">${hyrdrationWeek[5].date}</td>
+    <td class="hydra-data">${hyrdrationWeek[5].numOunces}</td>
+  </tr>
+  <tr>
+    <td class="hydra-data">${hyrdrationWeek[6].date}</td>
+    <td class="hydra-data">${hyrdrationWeek[6].numOunces}</td>
+  </tr>
+</table>`;
+  } else {
```
Comment (piperjarrett) [inline src/scripts.js:258]: good sad path in case they choose an early date!

**Review by piperjarrett — COMMENTED** [review]
Looks good! Just one change in the CSS but Nicole and I can work on that since you are traveling!

**Review by forsethnico — APPROVED** [review]
Looks great!

**Pull request merged**